templates/zerver/features.html

```diff
@@ -301,7 +301,7 @@
                     <div class="block">
                         <h4>{% trans %}Desktop apps{% endtrans %}</h4>
                         <p>{% trans %}Prefer Zulip in its own window and rich, OS-level notifications?
-                          Enjoy <a href="/apps">Zulip on your desktop</a>.{% endtrans %}</p>
+                          Enjoy <a href="https://zulipchat.com/apps">Zulip on your desktop</a>.{% endtrans %}</p>
```

Comment:
Done. I made it just use ZILENCER_ENABLED. You also mentioned REALMS_HAVE_SUBDOMAINS, but I'm not sure how the interaction goes -- do you think the right condition to use should involve that too?